### PR TITLE
add geneva action to list backup blobs

### PR DESCRIPTION
**Pull request by jim-minter**
```release-note
* add geneva action to list backup blobs
```

### CODE REVIEW

File: pkg/fakerp/routes.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ func (s *Server) SetupRoutes() {
 	s.router.Put(s.basePath, s.handlePut)
 	s.router.Get(filepath.Join("/admin", s.basePath), s.handleGet)
 	s.router.Put(filepath.Join("/admin", s.basePath), s.handlePut)
+	s.router.Put(filepath.Join("/admin", s.basePath, "/listBackups"), s.handleListBackups)
```
Comment (charlesakalugwu) [inline pkg/fakerp/routes.go]: This should be changed to a 
```suggestion
	s.router.Get(filepath.Join("/admin", s.basePath, "/listBackups"), s.handleListBackups)
```

File: pkg/cluster/etcdrestore.go [hunk 2]
```diff
@@ -2,25 +2,26 @@ package cluster
...
-	exists, err := blob.Exists()
+	resp, err := etcdContainer.ListBlobs(storage.ListBlobsParameters{})
 	if err != nil {
-		return err
+		return nil, err
 	}
-	if !exists {
-		return fmt.Errorf("Blob %s does not exist", blobName)
+	backups := make([]string, 0, len(resp.Blobs))
+	for _, blob := range resp.Blobs {
```
Comment (charlesakalugwu) [inline pkg/cluster/etcdrestore.go]: it might also be worth returning `blob.Properties.LastModified`

File: pkg/fakerp/routes.go (hunk 1 above)
Comment (jim-minter) [reply pkg/fakerp/routes.go]: good catch thanks

File: pkg/cluster/etcdrestore.go (hunk 2 above)
Comment (jim-minter) [reply pkg/cluster/etcdrestore.go]: done, thanks

**Pull request merged**